README.md

```diff
@@ -67,6 +67,16 @@ This connects a client that pings the `HelloWorker` and `DiceWorker` every few s
 ## Logs location
 The logs are stored in `logs` subdirectory of the location where your `spatialos.json` file is. For example, if my `spatialos.json` file is in `/workspace/project/SpatialOS`, the SpatialOS logs will go to `/workspace/project/SpatialOS/logs`.
 
+## Known Issues
+* `spatial local worker launch` is not supported.
```

Comment:
`spatial local worker launch` is not supported. You will need to launch workers manually through the steps indicated in (link to the part of the doc above)

?

Reply:
Re: How do we start workers in the future: Yes, users will need to manually start them.
Re: Should this be a known issue: Moved this note into a FAQ which will later become part of our migration guide.